daskhub-rhg/Chart.yaml

```diff
@@ -6,15 +6,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1
+version: 0.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.0.1"
+appVersion: "0.1.0"
 
 dependencies:
   - name: daskhub
-    version: "4.5.7"
+    version: "4.5.8"
```

Comment:
Looking at https://github.com/dask/helm-chart/releases, it looks like they changed their release style from semantic versioning to a versioning code based on date. So the newest version is  `2021.3.0` and not `4.5.8`.

That change isn't really noted anywhere, I'm afraid.